Sources/ProjectDescription/SchemeDiagnosticsOptions.swift

```diff
@@ -0,0 +1,35 @@
+import Foundation
+
+/// Options to configure scheme diagnostics for run and test actions.
+public struct SchemeDiagnosticsOptions: Equatable, Codable {
+    /// Enable the address sanitizer
+    public var addressSanitizerEnabled: Bool
+
+    /// Enable the detect use of stack after return of address sanitizer
+    public var detectStackUseAfterReturnEnabled: Bool
+
+    /// Enable the thread sanitizer
+    public var threadSanitizerEnabled: Bool
+
+    /// Enable the main thread cheker
+    public var mainThreadCheckerEnabled: Bool
+
+    /// Enable thread performance checker
+    public var performanceAntipatternCheckerEnabled: Bool
+
+    public static func options(
```

Comment:
Should this be `schemeDiagnosticOptions` or `diagnosticsOptions`? I think `options` is a bit too generic 🤔 

Although, in most cases it will be `diagnosticsOptions: .options(...)`, so maybe it's fine, I don't feel too strongly about this one.

Reply:
In other places we do `.options` so I leaned on the consistency side eliminating some redundancy.